core/web/resolver/vrf.go

```diff
@@ -67,14 +69,19 @@ func (r *VRFKeySuccessResolver) Key() VRFKeyResolver {
 
 type VRFKeyPayloadResolver struct {
 	key vrfkey.KeyV2
-	err error
+	NotFoundErrorUnionType
 }
 
 func NewVRFKeyPayloadResolver(key vrfkey.KeyV2, err error) *VRFKeyPayloadResolver {
-	return &VRFKeyPayloadResolver{
-		key: key,
-		err: err,
+	var e NotFoundErrorUnionType
+
+	if err != nil {
```

Comment:
Also here.

Reply:
same reason as stated before on why this needs to happen